### DISCUSSION TITLE
how to change the dialog control text

### DISCUSSION THREAD

**Discussion by radz1** [wixtoolset/issues#6342] [Questions] [answered]
Hi,
 i am using WixUI_InstallDir.wxs file. i need to change the text of browse folder button from 'Change...' to 'Something else'. how can we do.
Thanks,

**Comment by radz1** [comment, marked as answer]
> Hi,
> i am using WixUI_InstallDir.wxs file. i need to change the text of browse folder button from 'Change...' to 'Something else'. how can we do.
> Thanks,

found the solution by overriding the text key.

**Comment by BMurri** [comment]
Add a WXL file, where you override the text key.
